js-exercises/balanced-braces/balancedBraces.js

```diff
@@ -0,0 +1,18 @@
+function balancedBraces(input) {
+  const stack = []
+  const bracePairs = {"{": "}", "[": "]", "(": ")"}
+  for (const charecter of input) {
```

Comment:
typo in variable name. Please consider using es-lint in your IDE as as multiple small problems like semi column missing could be resolved automatically.

Reply:
Fixed using ESLint